client/components/sidebar.vue

```diff
@@ -527,6 +535,16 @@
         this.currentHighlightKey = highlightKey;
         this.layoutCommentsAfterRender();
       },
+
+      onShowDeletionDialog(comment) {
+        this.commentToDelete = comment;
+        this.$refs.commentDeletionDialog.show = true;
+        this.dialogType = comment.isMain ? 'thread' : 'single';
```

Comment:
Can we make it so that it is thread only if it is a main comment and if there are more than one comment in a thread (so if there are no replies, we should say to the user only "do you want to delete this comment", but if there are replies, then we should say to the user that deletion will delete whole thread).